master/buildbot/steps/source/git.py

```diff
@@ -570,35 +567,32 @@ def applyAlready(res):
         d.addCallback(applyAlready)
         return d
 
+    @defer.inlineCallbacks
     def _sourcedirIsUpdatable(self):
         if self.slaveVersionIsOlderThan('listdir', '2.16'):
-            d = self.pathExists(self.build.path_module.join(self.workdir, '.git'))
+            git_path = self.build.path_module.join(self.workdir, '.git')
```

Comment:
No, it's not. the line was getting a bit long and I wanted to split it... Plus, giving things a name can help readability and make the lines easier to read.